src/components/calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Button from './button';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="display">0</div>
+    <div className="button-container">
+      <Button label="AC" />
+      <Button label="+/-" />
+      <Button label="%" />
+      <Button label="/" />
+      <Button label="7" />
+      <Button label="8" />
+      <Button label="9" />
+      <Button label="x" />
+      <Button label="4" />
+      <Button label="5" />
+      <Button label="6" />
+      <Button label="-" />
+      <Button label="1" />
+      <Button label="2" />
+      <Button label="3" />
+      <Button label="+" />
+      <Button label="0" />
+      <Button label="." />
+      <Button label="=" />
+    </div>
+  </div>
+);
```

Comment:
- please make sure to stick as much as possible to the given design, which includes:
   - [ ] Making the "0" button span for two columns instead of just one.
   - [ ] Following the same color pattern used throughout the given template.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/DianaBeki/math-magicians/assets/62290545/342a3a57-1ad1-40e8-9676-c16b7f682ccf) | ![image](https://github.com/DianaBeki/math-magicians/assets/62290545/a6e7d9fd-8d1e-4ced-b6b7-29e307742369) |